docs/FRSKY_R9.md

```diff
@@ -189,4 +193,18 @@ The ST-Link connection is made as described in the ExpressLRS docs:
 - R9M module: https://www.expresslrs.org/1.0/quick-start/tx-r9m/#flashing-using-stlink
 - R9MX receiver: https://www.expresslrs.org/1.0/quick-start/rx-stlink/
 
+Use the non-"elrs-bl" firmware versions, which can be identified by the ".hex" extension and that the label "elrs-bl" is not contained in the firmware file name. 
+
 In these docs it is suggested to download and use the "ST-LINK Utility" software. This software is pretty outdated (NRND = not recommended for new designs), and the new recommended tool is "STM32CubeProgrammer".
+
+### Update via System Bootloader ###
+
+When the R9MX or R9MM receivers have been flashed with the non-"elrs-bl" firmware version via ST-Link, then they can be upgraded also by invoking the STM32 system bootloader, as follows:
```

Comment:
Suggestion: "When the R9MX or R9MM receivers have been flashed with the non-"elrs-bl" firmware version via ST-Link, they can be upgraded by invoking the STM32 system bootloader, which can be done as follows:

Reply:
THX! applied.